clients/pkg/promtail/client/client.go

```diff
@@ -93,6 +96,12 @@ func NewMetrics(reg prometheus.Registerer, streamLagLabels []string) *Metrics {
 		Help:      "Number of times batches has had to be retried.",
 	}, []string{HostLabel})
 
+	m.dropCount = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: "promtail",
+		Name:      "dropped_lines_total",
```

Comment:
I think there's already a metric with a very similar name. Could you double check that this isn't the case? If so, consider picking a different name.